cmd/crc/cmd/start.go

```diff
@@ -83,6 +83,14 @@ func runStart(arguments []string) (*machine.StartResult, error) {
 	return client.Start(startConfig)
 }
 
+func runOrSkipPreflightChecks(skip bool) error {
+	if skip {
+		logging.Info("All preflight checks are skipped")
+		return nil
+	}
+	return preflight.StartPreflightChecks(config)
+}
+
```

Comment:
This function is duplication for start and handler, better to have it part of preflight package itself and make `StartPreflightChecks` less visible. you can take `skip and config` as parameter for this one.

Reply:
Changed!